Doc/library/subprocess.rst

```diff
@@ -268,10 +268,12 @@ default values. The arguments that are most commonly needed are:
    :data:`PIPE` indicates that a new pipe to the child should be created.
    :data:`DEVNULL` indicates that the special file :data:`os.devnull` will
    be used.  With the default settings of ``None``, no redirection will occur;
-   the child's file handles will be inherited from the parent.
-   Additionally, *stderr* can be :data:`STDOUT`, which indicates that the
-   stderr data from the child process should be captured into the same file
-   handle as for *stdout*.
+   the child's file handles will be inherited from the parent for console
+   applications, if the child can connect to the console.  Otherwise it will
+   allocate a new console. For non-console applications, the window manager
+   and graphical shell will determine behavior.  Additionally, *stderr* can be
```

Comment:
I'd suggest avoid changing the original sentence structure starting from 'Additionally' so it'll a bit easier to review the diffs and changes, since the sentences are identical from that point onward.

Reply:
I understand the point, start a new line at "Additionally". There are two instances of this dcoument section, lines 272 and 492 in the existing code. For 272, it would be easy to insert as you noted, but the "Additionally" on line 492 is not at a line break. What should be done about that one?

Reply:
Same, try to minimize changes to untouched lines.  It is fine if one line is a bit too long, or if you move its end to a new line that’s a bit short.  It doesn’t change the output but keeps git log / annotate / etc useful 🙂

Reply:
Changes made, did I understand and implement correctly?

Thanks!

Reply:
Not fully, but I completed the change!